codalab/rest/users.py

```diff
@@ -73,6 +76,56 @@ def fetch_user(user_spec):
     return allowed_user_schema()().dump(user).data
 
 
+@delete('/users', apply=AuthenticatedPlugin())
+def delete_user():
+    """Fetch user ids"""
+    user_ids = get_resource_ids(request.json, 'users')
+
+    request_user_id = request.user.user_id
+
+    if request_user_id != local.model.root_user_id:
+        abort(httplib.UNAUTHORIZED, "Only root user can delete other users.")
+
+    for user_id in user_ids:
+        if user_id == local.model.root_user_id:
+            abort(httplib.UNAUTHORIZED, "Cannot delete root user.")
+        user = local.model.get_user(user_id=user_id)
+        if user is None:
+            abort(httplib.NOT_FOUND, "User %s not found" % user_id)
+
+        """
+        Check for owned bundles, worksheets, and groups.
+        If any are found, then do not allow user to be deleted.
+        """
+        error_message = ""
+
+        bundles = local.model.batch_get_bundles(owner_id=user_id)
+        if bundles is not None and len(bundles) > 0:
+            bundle_uuids = [bundle.uuid for bundle in bundles]
+            error_message += "User %s owns bundles, can't delete. UUIDs: %s \n" % \
```

Comment:
clarify: "can't delete user"

Remove extra space before newline.

I'd also build up a list of error messages and then join with '\n' at the end - which is semantically better.